stable/elasticsearch-exporter/templates/servicemonitor.yaml

```diff
@@ -18,6 +18,7 @@ spec:
     honorLabels: true
     port: "{{ .Values.service.httpPort }}"
     path: {{ .Values.web.path }}
+    scheme: http
```

Comment:
Shouldn't  this be configurable?

Reply:
@vsliouniaev The elasticsearch-exporter exposes only http endpoint for prometheus metrics. Is there any case when it is exposed over https or other scheme?

Reply:
@vsliouniaev Have we got any plan to expose it via https??

Reply:
I'm not a maintainer for this chart so it was more a suggestion, since I'm working with prometheus-operator

Reply:
Let's keep it like this for now since https://github.com/helm/charts/blob/master/stable/elasticsearch-exporter/templates/service.yaml#L17 is hardcoded as well.